README.md

```diff
@@ -12,19 +12,98 @@ other components of the system, including the Istio mixer and the Istio proxy me
 If you have a question about the Istio Manager or have a problem using it, please
 [file an issue](https://github.com/istio/manager/issues/new).
 
-## Build instructions ##
+## Setting up a local dev environment ##
+
+_Note:_ This section applies to Mac and Windows users only.
+
+### Pre-requisites ##
+
+- Setup Go 1.7.5+ on your host machine
+- Clone this repository
+- Install [Virtualbox](https://github.com/kubernetes/minikube/releases)
+- Install [Minikube](https://github.com/kubernetes/minikube/releases)
+- Install [Vagrant](https://www.vagrantup.com/downloads.html)
+- Install [kubectl](https://kubernetes.io/docs/user-guide/prereqs/)
+
+### 1. Start Minikube
+
+    minikube config set kubernetes-version v1.5.2
+    minikube start
+    
+Copy the kube config file to the platform/kube directory and update the paths
+
+    cp ~/.kube/config platform/kube/
```

Comment:
We should use symlink. The method to debug k8s is to use kubectl and unless you symlink, you'll be lost which cluster you are using. I have many clusters in kubectl as many other users do.